packages/common/src/interaction.ts

```diff
@@ -237,13 +235,30 @@ export interface ExecutionRunRef
 
 export const ExecutionRunRefSelect = "-result -parameters -result_schema -prompt";
 
-export interface InteractionExecutionConfiguration {
+export enum ConfigModes {
+    OVERWRITE = "OVERWRITE",
+    RUN_ONLY = "RUN_ONLY",
+    INTERACTION_ONLY = "INTERACTION_ONLY"
```

Comment:
nit: would it be easier to understand if we call them

* RUN_AND_INTERACTION_CONFIG
* RUN_CONFIG_ONLY
* INTERACTION_CONFIG_ONLY

Asking because I feel like that it's hard to understand what is "overwrite" compared to the 2 other options. Mentioning both run and interaction makes me feel easier to connect with other elements in the enum. And since it's about the usage of configuration, I feel like that including the word "config" in the mode makes it easier to understand what is being addressed as well.

Reply:
Makes sense to me, change has been made.